packages/ember-metal/lib/tags.js

```diff
@@ -19,11 +24,23 @@ export function tagFor(object, _meta) {
   }
 }
 
+function K() {}
+function ensureRunloop() {
+  if (!run) {
+    run = require('ember-metal/run_loop').default;
```

Comment:
This should be able to move to the top now that this isn't importing ember-glimmer